lib/library.go

```diff
@@ -327,7 +327,7 @@ func InitJail(js *C.char) {
 //DEPRECATED in favour of CreateAndInitCell.
 //export Parse
 func Parse(chatID *C.char, js *C.char) *C.char {
-	res := statusAPI.JailParse(C.GoString(chatID), C.GoString(js))
```

Comment:
Could you also clean up `StatusAPI.JailParse` as it's not needed? We also don't need `Jail.Parse` anymore.

Reply:
Oh, scratch that. Let's leave it for now. status-react is not ready yet to remove it.